src/main/java/com/frostwire/jlibtorrent/SessionManager.java

```diff
@@ -1119,6 +1119,10 @@ public void run() {
         t.start();
     }
 
+    public TorrentStats trackStats(TorrentHandle torrentHandle, long samplingIntervalInMs, long maxHistoryInMs) {
```

Comment:
I'm just thinking about guys that could be seeding thousands of torrents, perhaps they might not want to have all these calculations.

In the case of FW, what I'm thinking of doing is to keep track of the last N clicked torrents in the TransferManager, so if the user is trying to compare the speeds between them, there will be no incomplete graphs, and I'll make N a number that depends on how it affects the UI performance. Perhaps something like N < 20